README.md

```diff
@@ -9,6 +9,10 @@ to their data as a TensorFlow graph. This is important as the user can then
 incorporate the exported TensorFlow graph into their serving model, thus
 avoiding skew between the served model and the training data.
 
```

Comment:
Perhaps we should also link to the Research Blog Post for TFT? @KesterTong what do you think? Any suggestions for location?

Reply:
In general I feel more cross-linking the better :) But here I will leave it to the esteemed reviewers.

Reply:
This README supercedes as a description of tf.Transform, so I think a link is not useful.

Reply:
I mean that this README supersedes the blog post.